pkg/webui/console/store/middleware/app.js

```diff
@@ -12,48 +12,42 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/* eslint-disable no-console */
-
 import { createLogic } from 'redux-logic'
 
 import * as user from '../actions/user'
-import * as _console from '../actions/console'
-import api from '../api'
-import * as accessToken from '../lib/access-token'
+import * as app from '../actions/app'
+import api from '../../api'
+import * as accessToken from '../../lib/access-token'
 
-const consoleLogic = createLogic({
-  type: _console.INITIALIZE,
+const consoleAppLogic = createLogic({
+  type: app.INITIALIZE,
```

Comment:
Idk might this be confusing? Still `_console` was not the best naming either, so I guess it doesn't matter.